backend/config/corpora-api.yml

```diff
@@ -492,11 +492,85 @@ paths:
         "404":
           $ref: "#/components/responses/404"
 
+  /v1/login:
+    get:
+      tags:
+        - authentication
+      summary: login to the system
+      operationId: corpora.lambdas.api.v1.authentication.login
+      responses:
+        "200":
+          $ref: "#/components/responses/200"
+        "401":
+          $ref: "#/components/responses/401"
+        "403":
+          $ref: "#/components/responses/403"
+
+  /v1/logout:
+    get:
+      tags:
+        - authentication
+      summary: logout of the system
+      operationId: corpora.lambdas.api.v1.authentication.logout
+      responses:
+        "200":
+          $ref: "#/components/responses/200"
+        "401":
+          $ref: "#/components/responses/401"
+        "403":
+          $ref: "#/components/responses/403"
+
+  /v1/userinfo:
+    get:
+      tags:
+        - authentication
+      summary: retrieve user information if the user is logged in
+      operationId: corpora.lambdas.api.v1.authentication.userinfo
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/userinfo"
+          $ref: "#/components/responses/200"
+        "401":
+          $ref: "#/components/responses/401"
+        "403":
+          $ref: "#/components/responses/403"
+
+  /v1/oauth2/callback:
+    get:
+      tags:
+        - authentication
+      summary: callback function from auth server after login
```

Comment:
can this also be used for logout?

Reply:
No, this is just for login.   The logout is handled in a simpler way.  It logs out of auth0, removes the cookie, then redirects back to "/".